### PR TITLE
Add endpoint /features/create-region 

### PR DESCRIPTION
**Pull request by pthomas76**
This adds the endpoint /features/create-region to add a new row to the database. Mandatory parameters in the POST request are name, namelsad and stusps.

### CODE REVIEW

**Comment by ankithkanderi24** [comment]
You should return a success code at the end of the code block

**Comment by pthomas76** [comment]
@ankithkanderi24 added it. everything else fine?

**Comment by ankithkanderi24** [comment]
Everything else looks great!

File: database/API.py [hunk 1]
```diff
@@ -43,4 +43,18 @@ def features_get_by_sno():
     for feature in session.scalars(stmt):
         result.append(feature.as_dict())
     
-    return result, 200
\ No newline at end of file
+    return result, 200
+
+@app.route('/features/create-region', methods=['POST'])
```
Comment (DXGatech) [inline database/API.py:48]: Suggestion (non-blocking): some of our functions have a docstring and this one does not. It is not necessary but I think docstrings can be very useful for a dev who hasn't worked on the backend yet

**Pull request merged**